main.js

```diff
@@ -0,0 +1,130 @@
+let isClicked = false;
+
+// eslint-disable-next-line no-unused-vars
```

Comment:
- [ ] As the previous Code Reviewer mentioned, `eslint` should not be disabled. Kindly remove that.